model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -690,7 +690,7 @@ case class WorkspaceListResponse(accessLevel: WorkspaceAccessLevel,
                                  public: Boolean
 )
 
-case class AzureManagedAppCoordinates(tenantId: UUID, subscriptionId: UUID, managedResourceGroupId: String)
+case class AzureManagedAppCoordinates(tenantId: UUID, subscriptionId: UUID, managedResourceGroupId: String, landingZoneId: Option[UUID] = None)
```

Comment:
I'm adding the optional LZ ID param to this model to keep this contained within the Azure concerns of a billing project creation request, rather than including it as a top level param. This will require a rawls model lib update.

Reply:
Do you need to add it to Swagger as well? Or should it stay out of Swagger since it's not for production use?

Reply:
It should stay out of swagger, this is an "internal use only" parameter.